package-lock.json

```diff
@@ -17,6 +17,11 @@
         "@types/babel-types": "*"
       }
     },
+    "abbrev": {
```

Comment:
There are lots of changes to this file. What are all these for?

Reply:
this is auto generated, should be fine with the package.json changes